crml/erc20-peg/src/lib.rs

```diff
@@ -166,8 +170,18 @@ decl_module! {
 			// otherwise there may be no liquidity on the Ethereum side of the peg
 			let token_address = Self::asset_to_erc20(asset_id);
 			ensure!(token_address.is_some(), Error::<T>::UnsupportedAsset);
-
-			let _imbalance = T::MultiCurrency::withdraw(&origin, asset_id, amount, WithdrawReasons::TRANSFER, frame_support::traits::ExistenceRequirement::KeepAlive)?;
+			let staking_currency = T::MultiCurrency::staking_currency();
+			if asset_id == staking_currency && Self::cennz_deposit_active() {
```

Comment:
had intended this var for use with original erc20 cennz pending a governance vote `cennz_deposit_active`
we wouldn't need it for wCENNZ
```suggestion
			if asset_id == staking_currency {
```